CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## Release 1.2.1
+This is a minor patch release.
+
+- Uploaded new cloud token screenshot
```

Comment:
Please classify the items in sections (e.g. Documentation) like in the previous releases